### DISCUSSION TITLE
additional informations for our seplos bms ...

### DISCUSSION THREAD

**Discussion by manznOnly** [syssi/esphome-seplos-bms#53] [Ideas]
Hy Syssi 

found this project from BrucePerens, I think he had decode much HEX Codes from Seplos Communication Protocoll 2 ... my C is very poor - but maybe it can help you!?

Take a look here:
https://github.com/BrucePerens/seplos_c/blob/main/library/data.c

br,
manzn

**Comment by syssi** [comment]
Good catch! The code will be helpful to understand the decoding of some error/protection bits.
